lib/matplotlib/pyplot.py

```diff
@@ -1038,115 +1038,125 @@ def subplot(*args, **kwargs):
+          plots but the ones on first column.
+        - If False, no y-axis will be shared amongst subplots.
+        - If a string must be one of "row", "col", "all", or "none".
+            - "all" has the same effect as True.
+            - "none" has the same effect as False.
+            - If "row", each subplot row will share a y-axis.
+            - If "col", each subplot column will share a y-axis. Note that if
+              ncols > 1 then the y tick labels won't be displayed on any of
+              the plots but the ones on the first column.
+
+    squeeze : bool, optional, default: True
+        - If True, extra dimensions are squeezed out from the returned axis
+          object:
+
+            - if only one subplot is constructed (nrows=ncols=1), the
+              resulting single Axis object is returned as a scalar.
+            - for Nx1 or 1xN subplots, the returned object is a 1-d numpy
+              object array of Axis objects are returned as numpy 1-d arrays.
+            - for NxM subplots with N>1 and M>1 are returned as a 2d arrays.
+
+        - If False, no squeezing at all is done: the returned axis object is
+          always a 2-d array containing Axis instances, even if it ends up
+          being 1x1.
+
+    subplot_kw : dict, optional
         Dict with keywords passed to the
-        :meth:`~matplotlib.figure.Figure.add_subplot` call used to
-        create each subplots.
+        :meth:`~matplotlib.figure.Figure.add_subplot` call used to create each
+        subplot.
 
-      *gridspec_kw* : dict
+    gridspec_kw : dict, optional
         Dict with keywords passed to the
-        :class:`~matplotlib.gridspec.GridSpec` constructor used to create
-        the grid the subplots are placed on.
+        :class:`~matplotlib.gridspec.GridSpec` constructor used to create the
+        grid the subplots are placed on.
 
-      *fig_kw* : dict
+    fig_kw : dict, optional
         Dict with keywords passed to the :func:`figure` call.  Note that all
         keywords not recognized above will be automatically included here.
 
-    Returns:
+    Returns
+    -------
+    fig : :class:`matplotlib.figure.Figure` object
 
-    fig, ax : tuple
+    ax : axis or array of axis objects.
+        ax can be either a single axis object or an array of axis objects if
+        more than one subplot was created.  The dimensions of the resulting
+        array can be controlled with the squeeze keyword, see above.
 
-      - *fig* is the :class:`matplotlib.figure.Figure` object
+    Examples
+    --------
+    First create some toy data:
 
-      - *ax* can be either a single axis object or an array of axis
-        objects if more than one subplot was created.  The dimensions
-        of the resulting array can be controlled with the squeeze
-        keyword, see above.
+    >>> x = np.linspace(0, 2*np.pi, 400)
+    >>> y = np.sin(x**2)
 
-    Examples::
+    Creates just a figure and only one subplot.
```

Comment:
I would go with imperative tense ("Create just ..."); the sharing sections use this tense as well. Also, this is the only one with a period at the end.